tests/chainer_tests/functions_tests/connection_tests/test_linear.py

```diff
@@ -41,138 +33,68 @@ def _to_noncontiguous(arrays):
     + [
         {'use_chainerx': True, 'chainerx_device': 'native:0'},
         {'use_chainerx': True, 'chainerx_device': 'cuda:0'},
-    ])
-class TestNonparameterizedLinear(unittest.TestCase):
+    ]
+)
+class TestNonparameterizedLinear(testing.FunctionTestCase):
 
     def setUp(self):
+        self.check_backward_options = {'atol': 1e-2, 'rtol': 1e-2}
+        self.check_double_backward_options = {'atol': 1e-2, 'rtol': 1e-2}
+        if self.x_dtype == numpy.float16:
```

Comment:
Do you still need these relaxations (for non ChainerX tests)?

Reply:
This relaxation is from the previous tests and I didn't try to narrow it.
See:
https://github.com/chainer/chainer/blob/afe903389d822583a5355e9d46e6766d048ebeb5/tests/chainer_tests/functions_tests/connection_tests/test_linear.py#L71-L72

Reply:
Sorry I missed that. Thanks.